package.json

```diff
@@ -7,7 +7,7 @@
 		"ui-for-watcher": {
 			"distDir": "dist/",
 			"optimize": false,
-			"sourceMap": false
+			"sourceMap": true
```

Comment:
Moet deze wel op true?

Reply:
Ja watcher is lokaal draaien dus wil je zoveel mogelijk de originele code zien. Dus de source map hebben.

Reply:
Ja want je wilt source maps in je dev omgeving